src/data/api/datatyper/apiOptions.ts

```diff
@@ -0,0 +1,22 @@
+import { FrontendEvent } from '../../../utils/logger';
+
+export const GEToptions = {
+    method: 'GET',
+    headers: {
+        'Nav-Consumer-Id': 'veilarbdetaljerfs',
+        Accept: 'application/json',
+        'Content-Type': 'application/json;charset=UTF-8'
+    }
+};
+
+export const createPOSToptions = (event: FrontendEvent) => {
```

Comment:
bare for å være ekstra fancy så er det ikke noe behov for å opprette en const bare for å returnere den:

```
export const createPOSToptions = (event: FrontendEvent) => ({
	method: 'POST',
	body: JSON.stringify(event),
	headers: {
		'Nav-Consumer-Id': 'veilarbdetaljerfs'
	}
})
```